CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+[cygnus-common][SQLBackendImpl] Add configuration option to persist errors in SQL sinks (#1928) 
```

Comment:
Similar to the existing functionality for PG?

Reply:
just make it optional by config

Reply:
Ok. NTC